server/src/domain/user/user.service.ts

```diff
@@ -75,14 +75,21 @@ export class UserService {
     return this.userCore.updateUser(auth.user, dto.id, dto).then(mapUser);
   }
 
-  async delete(auth: AuthDto, id: string): Promise<UserResponseDto> {
+  async delete(auth: AuthDto, id: string, options: DeleteUserDto): Promise<UserResponseDto> {
     const user = await this.findOrFail(id, {});
     if (user.isAdmin) {
       throw new ForbiddenException('Cannot delete admin user');
     }
 
     await this.albumRepository.softDeleteAll(id);
 
+    if (options && options.force) {
```

Comment:
Options can't be null anymore so I would just add `const { force } = dto` at the top